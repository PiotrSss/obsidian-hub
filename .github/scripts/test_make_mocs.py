# TODO Figure out how to move this to a tests directory, abd still import make_moc

from approvaltests.approvals import verify

from make_mocs import make_moc_for_directory, make_moc_for_sub_directories


def test_output_for_files():
    directory = '01 - Topic 1/01.02 Subtopic'
    files = ['README.md', 'publish.css', 'logo.svg', 'File 1.md', 'File 2.md']
    result = make_moc_for_directory(directory, files)
    verify(result)


def test_output_for_sub_directories():
    directory = '01 - Community'
    sub_directories = ['Authors - Persons', 'Events', 'Obsidian Roundup', 'Video Channels']
    result = make_moc_for_sub_directories(directory, sub_directories)
    verify(result)
