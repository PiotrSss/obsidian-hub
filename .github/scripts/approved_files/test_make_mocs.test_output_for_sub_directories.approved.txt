-  [[Authors - Persons]]
-  [[Events]]
-  [[Obsidian Roundup]]
-  [[Video Channels]]
